src/app/clusters/door-lock-server/door-lock-server.h

```diff
@@ -121,7 +121,7 @@ class DoorLockServer
     bool SetOneTouchLocking(chip::EndpointId endpointId, bool isEnabled);
     bool SetPrivacyModeButton(chip::EndpointId endpointId, bool isEnabled);
 
-    bool TrackWrongCodeEntry(chip::EndpointId endpointId);
+    bool HandleWrongCodeEntry(chip::EndpointId endpointId);
```

Comment:
Needs documentation (the API docs should be here, not next to the impl).

Reply:
done